.lintstagedrc

```diff
@@ -0,0 +1,6 @@
+{
+  "**/*.+(ts|tsx)": [
```

Comment:
this does not match the config in `package.json` (i.e. we run it for more file formats : https://github.com/roam-unofficial/roam-toolkit/blob/master/package.json#L14)